robottelo/test.py

```diff
@@ -170,11 +171,16 @@ def setUpClass(cls):  # noqa
     def setUp(self):  # noqa
         """We do want a new browser instance for every test."""
         if self.driver_name.lower() == 'firefox':
-            self.browser = webdriver.Firefox()
+            self.browser = webdriver.Firefox(
+                firefox_binary=webdriver.firefox.firefox_binary.FirefoxBinary(
+                    self.driver_binary
+                ))
         elif self.driver_name.lower() == 'chrome':
-            self.browser = webdriver.Chrome()
+            self.browser = webdriver.Chrome() if self.driver_binary is None \
```

Comment:
`\` should not be used.

And having an if/else block is more readable than a ternary operator in this case because it needs more than one line.